math/minuit/src/TFitter.cxx

```diff
@@ -137,8 +137,12 @@ void TFitter::GetConfidenceIntervals(Int_t n, Int_t ndim, const Double_t *x, Dou
    Double_t c=0;
 
    Double_t *matr = GetCovarianceMatrix();
-   if (!matr)
+   if (!matr){
+      delete [] grad;
```

Comment:
I agree with Vassil.  Since we are touching (looking at) this code, it makes sense to make related improvements (especially when they are straight-forward) .. and indeed setting pointer to nullptr after deletion is somewhat of requirement.  Thanks.